docs/plugins/rate_limit.md

```diff
@@ -0,0 +1,45 @@
+# Rate Limiting
+
+Rate limit how many HTTP requests a developer can make in a given period of seconds, minutes, hours, days, months or years. If the API has no authentication layer, the Client IP address will be used, otherwise the Consumer will be used if an authentication plugin has been configured.
+
+## Configuration
+
+Configuring the plugin is straightforward, you can add it on top of an API by executing the following request on your Janus server:
+
+| Configuration | Description                                                                                                                                                                                                                                            |
+|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| limit         | Defines the limit rule for the proxy. i.e. "10-S", means 10 req/s                                                                                                                                                                                      |
```

Comment:
I could not find format for this field, only one sample in description for `10 req/s`. How do I do `15 req/min`, `100 req/h`, `10000 req/day`? If this supports only `req/s` then `-S` suffix is redundant, if it supports all types of limits that I mentioned - we need a description for this or a link to a format description.